README.md

```diff
@@ -105,11 +105,11 @@ After you're finished please remove all the comments and instructions!
 
 <!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"> Will be added later</a>
+## 🚀 Live Demo <a name="live-demo"> https://sajibarafatsiddiqui.github.io/project_leaderboard/dist/</a>
```

Comment:
- The MIT license link in your README seems broken. Kindly fix it for a more professional README 😉.

Reply:
The MIT license link has been added. Thanks